homeassistant/components/tplink_omada/translations/en.json

```diff
@@ -1,39 +1,40 @@
 {
```

Comment:
Seems like you accidentally committed this change. pre-commit auto changed that file.

Reply:
Yes I probably got confused by the name, either it has been a merge error. It should be fixed now